integreatly-mobile-security-service/0.4.1/mobile-security-service-operator.v0.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,184 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples:
+      '[{"apiVersion":"mobile-security-service.aerogear.org/v1alpha1","kind":"MobileSecurityServiceDB","metadata":{"name":"mobile-security-service-db"},"spec":{"containerName":"database","databaseMemoryLimit":"512Mi","databaseMemoryRequest":"512Mi","databaseName":"mobile_security_service","databaseNameParam":"POSTGRESQL_DATABASE","databasePassword":"postgres","databasePasswordParam":"POSTGRESQL_PASSWORD","databasePort":5432,"databaseStorageRequest":"1Gi","databaseUser":"postgresql","databaseUserParam":"POSTGRESQL_USER","image":"centos/postgresql-96-centos7","size":1}},{"apiVersion":"mobile-security-service.aerogear.org/v1alpha1","kind":"MobileSecurityService","metadata":{"name":"mobile-security-service"},"spec":{"accessControlAllowCredentials":"false","accessControlAllowOrigin":"*","clusterProtocol":"https","configMapName":"mobile-security-service-config","containerName":"application","databaseHost":"mobile-security-service-db","databaseName":"mobile_security_service","databasePassword":"postgres","databaseUser":"postgresql","image":"quay.io/aerogear/mobile-security-service:0.2.2","logFormat":"json","logLevel":"info","memoryLimit":"128Mi","memoryRequest":"64Mi","oAuthContainerName":"oauth-proxy","oAuthImage":"docker.io/openshift/oauth-proxy:v1.1.0","oAuthMemoryLimit":"64Mi","oAuthMemoryRequest":"32Mi","oAuthResourceCpu":"10m","oAuthResourceCpuLimit":"20m","port":3000,"resourceCpu":"10m","resourceCpuLimit":"20m","routeName":"route","size":1}},{"apiVersion":"mobile-security-service.aerogear.org/v1alpha1","kind":"MobileSecurityServiceApp","metadata":{"name":"mobile-security-service-app"},"spec":{"appId":"appid.example.com","appName":"example-appname"}},{"apiVersion":"mobile-security-service.aerogear.org/v1alpha1","kind":"MobileSecurityServiceBackup","metadata":{"name":"mobile-security-service-backup"},"spec":{"awsAccessKeyId":"example-awsAccessKeyId","awsS3BucketName":"example-awsS3BucketName","awsSecretAccessKey":"example-awsSecretAccessKey","databaseVersion":"9.6","image":"quay.io/integreatly/backup-container:1.0.8","productName":"mss","schedule":"0
+      0 * * *"}}]'
+    capabilities: Basic Install
+    categories: Security
+    description: >-
+      This is the server component of the AeroGear Mobile Security Service. It is a RESTful API that allows developers to view, enable and disable specific versions of applications on demand, with the information stored in a PostgreSQL database.
+    containerImage: mobile-security-service-operator
```

Comment:
This is only metadata, but should still point to a real image e.g. `quay.io/aerogear/mobile-security-service:v0.4.1`